projects/cdk/schematics/ng-update/v4/tests/schematic-textfield-controller.spec.ts

```diff
@@ -53,6 +53,12 @@ const TEMPLATE_BEFORE = `
         tuiTextfield
         type="email"
     />
+    <tui-data-list-wrapper
+        *tuiDataList
+        tuiMultiSelectGroup
+        [items]="items | tuiFilterByInputWith : matcher"
```

Comment:
`tuiFilterByInputWith` was used for stringify not for matcher. Now it gets stringify from host, so we should replace:
`xxx | tuiFilterByInputWith: yyy` with `xxx | tuiFilterByInput`